### ISSUE TITLE
notifyDataSetChanged is always scrolling to first entry

### ISSUE THREAD

**Issue by luggi2905** [yarolegovich/DiscreteScrollView#106]
Hello,

first of all, thanks a lot for your library. My issue is not a real issue since everything is working fine. But in this part of your code within InfiniteScrollAdapter.DataSetChangeDelegate you declare the methode:
```
public void onChanged() {
    resetRange(0);
    notifyDataSetChanged();
}
```
Within my code the amount or the order of the entries is not changing. 
When you click on one element in the ScrollView another window opens and some property of the element changes. If you go back, the look of the shown entry should be different (according to the changed property). To always have the right look, I load the current set everytime the user resumes to the activity with the ScrollView. But I want to display the last shown entry to the user. Do you understand my problem?

So could you change this part of the code to the following:

```
public void onChanged() {
    resetRange(currentRangeStart);
    notifyDataSetChanged();
}
```

**Comment by JyothishJohnson** [comment]
Yep, the view is scrolling back to the first element when it returns back from another activity. Please change this.

**Comment by mathemandy** [comment]
@JyothishJohnson @luggi2905  were you guys able to find a work around for this ?
i am facing the same issue here 🚨

**Comment by jjhesk** [comment]
I have forked the lib and have my own version so i think i can fix this bug. I faced the same issue too. https://github.com/HKMOpen/DiscreteScrollView

**Comment by JyothishJohnson** [comment]
@mathemandy Yep..actually I was loading the data from the network, so after loading the data I am calling notifyDataSetChanged() on the adapter. After this its working fine for me.

**Comment by mathemandy** [comment]
@jjhesk  i tried using the dependency on your forked version but gradle cannot seem to find it

**Comment by yarolegovich** [comment]
In `1.4.8` calling notifyItemChanged/notifyItemRangeChanged won't scroll the view to the first item.
